### DISCUSSION TITLE
Manage client on command line?

### DISCUSSION THREAD

**Discussion by Vichman67** [yuezk/GlobalProtect-openconnect#194] [Q&A]
Hello,

Thanks a million for your development. It saved my day under Ubuntu 22.04!

I'd like to know if there is any way to manage basic options under command shell. I mean, I wrote a simple bash script to connect to my VPN using the command '_/usr/bin/gpclient [MyVPN_URL] --now_' and I'd like to get the commands for:

1. Getting the status (connected / disconnected) of the VPN connection. Similar to PaloAlto CLI GloblaProtect '_globalprotect show --status_' command.

2. Disconnecting from the VPN. Similar to PaloAlto CLI GloblaProtect '_globalprotect disconnect_' command.

Any ideas appart from launching ifconfig command or killing /usr/bin/gpclient and /usr/sbin/openconnect processes?

TIA!

**Comment by yuezk** [comment]
Good suggestions! But I'm afraid I don't have enough time to implement it.

**Comment by Vichman67** [comment]
I understand it. Anyway, thanks for your hard work!
